apps/mobile/src/components/ProfileView/Tabs/ProfileViewFeaturedTab.tsx

```diff
@@ -56,7 +54,9 @@ export function ProfileViewFeaturedTab({ queryRef }: ProfileViewFeaturedTabProps
   return (
     <View style={contentContainerStyle}>
       <Tabs.FlashList
-        estimatedItemSize={300}
+        getItemType={(item) => item.kind}
+        keyExtractor={(item) => item.key}
+        estimatedItemSize={100}
```

Comment:
should have the `getItemType` and `keyExtractor` on all of these. `estimatedItemSize` helps with funky mounting behavior